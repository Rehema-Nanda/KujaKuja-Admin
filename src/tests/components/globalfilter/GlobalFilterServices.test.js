import React from "react";
import { shallow } from "enzyme";
import GlobalFilterServices from "../../../components/globalfilter/GlobalFilterServices";

describe("<GlobalFilterServices />", () => {
    const testProps = {
        serviceTypes: [{ name: "test", id: "1" }],
    };
    const wrapper = shallow(<GlobalFilterServices {...testProps} />);

    it("renders correctly", () => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
