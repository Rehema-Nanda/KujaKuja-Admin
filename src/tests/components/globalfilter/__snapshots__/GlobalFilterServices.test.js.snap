// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalFilterServices /> renders correctly 1`] = `
"<div>
  <div className=\\"map-controls-service-container\\">
    <GlobalFilterServiceButton id=\\"1\\" tipId=\\"serviceType1\\" iconoff={{...}} iconon={{...}} name=\\"test\\" toggleServiceType={[undefined]} selectedServiceTypes={[undefined]} />
  </div>
</div>"
`;
